Utility function preferring dirty buffer contents over saved file contents (#1341)
confluentinc/vscode: CHANGELOG.md

- New "Delete Schema Version" context menu action for schemas to drive either soft or hard deletions
  of single schema versions. If the final schema within a subject is deleted, the subject will no
  longer exist.

> - Actions using file contents chosen from open editor list such as schema upload or produce messages will now prefer to use the possibly unsaved editor buffer contents over any possibly stale on-disk contents. Unsaved documents may now be submitted.

Reviewer: Slight rewording for clarity?
```suggestion
- Unsaved file contents can now be used for schema upload or message producing actions without the need to save the file first.
```
